test/Microsoft.IdentityModel.Tokens.Tests/CryptoProviderFactoryTests.cs

```diff
@@ -47,13 +47,13 @@ public void CreateAndReleaseSignatureProviders(SignatureProviderTheoryData theor
                 {
                     var disposeCalled = GetSignatureProviderIsDisposedByReflect(signatureProvider);
```

Comment:
Should we add a similar check here that `_keyedHashObjectPool` was cleared too?

Reply:
@pmaytak @keegan-caruso did you mean for the Symmetric case?
KeyedHash is only for Symmetric signatures.

Reply:
Actually I meant for both - `_keyedHashObjectPool` in SymmetricSignatureProvider and `_asymmetricAdapterObjectPool` in AsymmetricSignatureProvider..